package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fh-mbaas-api",
-  "version": "5.13.1",
+  "version": "5.13.2",
```

Comment:
Yes, I'm all for improvements like that. What is a real release? Can this change be released? It wouldn't be included as the default in a version of the platform or templates until the next platform release, but maybe it's useful to @evanshortiss before that?